### DISCUSSION TITLE
Add an option to enable users to boot without updating the software.

### DISCUSSION THREAD

**Discussion by Lancer907** [vladmandic/sdnext#619] [Ideas]
Thank you for your work! 
Would it be possible to add an option to enable users to launch the software without updating (git pull). 
Because of the network environment in China, frequent updates will lead to a very long startup time and many error messages.

**Comment by vladmandic** [comment]
it already exists: `--noupdate`

**Reply by Lancer907** [reply]
Could you show me where to add this argument？ I know that I can add somthing like "--autolaunch" at webui-user.bat in A1111 WebUI., but I can't find the place here.

**Reply by tkiikt** [reply]
see this: #323 and other flags: #99
